examples/tabs.rs

```diff
@@ -75,6 +124,37 @@ impl App {
     pub fn previous(&mut self) {
         self.selected_tab = self.selected_tab.previous();
     }
+
+    fn render_tabs(&self, area: Rect, buf: &mut Buffer) {
```

Comment:
Split App into two impl sections, this part should be below `impl Widget for App {}`